src/requests/session.ts

```diff
@@ -144,6 +145,41 @@ export async function getSessionPassport(
   return response?.lowcal_sessions_by_pk?.passport || null;
 }
 
+export async function getSessionMetadata(
+  client: GraphQLClient,
+  sessionId: string,
+): Promise<SessionMetadata | null> {
+  const response: Record<"lowcal_sessions_by_pk", SessionMetadata> =
+    await client.request(
+      gql`
+        query GetSessionById($id: uuid!) {
+          lowcal_sessions_by_pk(id: $id) {
+            id
+            createdAt: created_at
+            submittedAt: submitted_at
+            flow {
+              id
+              slug
+              publishedFlows: published_flows(
+                order_by: { created_at: desc }
+                limit: 1
```

Comment:
note: I'm currently just pulling the latest published flow, _not_ a version that's actually meaningful to this session. 

I realised that the `publishedFlowId` for a session can change over the course of it's active "lifespan", so do we want to:
- pull the `publishedFlowId` based on `submittedAt` (or `createdAt` or last `reconciliation_request` for this session ??)
- only include `flowId`

Reply:
Let's drop `publlishedFlowId` :+1: 

This is queued up over here: https://github.com/theopensystemslab/digital-planning-data-schemas/pull/57